releases/tickets/v23_0_0.rst

```diff
@@ -127,67 +163,81 @@ Tickets merged before w_2021_40
 - `DM-28698 <https://jira.lsstcorp.org/browse/DM-28698>`_:  Restructure image formatter relationships [obs_base, obs_lsst]
 - `DM-28716 <https://jira.lsstcorp.org/browse/DM-28716>`_:  Add pybind wrapper to afw archive that handles adding set of HDUs [afw]
 - `DM-28794 <https://jira.lsstcorp.org/browse/DM-28794>`_:  Defects instantiation creates undefined/infinite bbox  [ip_isr]
-- `DM-28803 <https://jira.lsstcorp.org/browse/DM-28803>`_:  Add capability to discard stars with NaN annuli [meas_algorithms]
-- `DM-28955 <https://jira.lsstcorp.org/browse/DM-28955>`_:  Allow passing `alardSigGauss` parameter to `modelPsfMatch` task at runtime [ip_diffim]
+- `DM-28803 <https://jira.lsstcorp.org/browse/DM-28803>`_:  Add capability to discard stars with NaN annuli [meas_algorithms, pipe_tasks]
+- `DM-28853 <https://jira.lsstcorp.org/browse/DM-28853>`_:  Improve WCS functionality within fake source injection [pipe_tasks]
+- `DM-28955 <https://jira.lsstcorp.org/browse/DM-28955>`_:  Allow passing \`alardSigGauss\` parameter to \`modelPsfMatch\` task at runtime [ip_diffim, meas_extensions_gaap]
```

Comment:
You may want to update your generator to convert single backticks to double backticks in reStructuredText rather than escaping them.

Reply:
Thanks for the hints. Should I escape `_` as well everywhere?

Reply:
Escaped all `_` as well. Backticks are converted to double backticks.